bugzilla.py

```diff
@@ -0,0 +1,12 @@
+import utils
+
+
+def get_signatures_from_bug(bug_id):
+    url = 'https://bugzilla.mozilla.org/rest/bug'
+    response = utils.get_with_retries(url, params={'id': bug_id})
+    response.raise_for_status()
+    return [str(sig)[2: len(sig) - 1].strip() for sig in response.json()['bugs'][0]['cf_crash_signature'].split('\r\n')]
```

Comment:
Can you do `sig.strip(' [@]')` instead of `str(sig)[2: len(sig) - 1].strip()`, just to be extra safe.